kostyor/resources/clusters.py

```diff
@@ -0,0 +1,36 @@
+from flask import request
+from flask.ext.restful import Resource, fields, marshal_with, abort
```

Comment:
Ah, _this_ is what is causing the warning.

Please fix.

https://travis-ci.org/sc68cal/Kostyor/builds/161383276#L572